crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -550,6 +562,36 @@ fn deserialize_cheat_target(inputs: &[Felt252]) -> (CheatTarget, usize) {
     }
 }
 
+fn serialize_call_entry_point(call_entry_point: &CallEntryPoint) -> Vec<Felt252> {
+    let mut output = vec![];
```

Comment:
What would you say to first initializing all the needed variables, and then creating the serialized vector at the end of the function in one go? Would make the contents of the vector more visible
```rust
    let mut output = Vec::new();
    output.push(Felt252::from(entry_point_type));
    output.push(call_entry_point.entry_point_selector.0.into_());
    output.extend(calldata);
    output.push(...);
```

Reply:
If it is more readable for you I can do that